code/game/objects/objs.dm

```diff
@@ -374,6 +374,16 @@
 /obj/proc/plunger_act(obj/item/plunger/P, mob/living/user, reinforced)
 	return
 
+/obj/attack_hand(mob/user)
+	if(user.a_intent == INTENT_HARM)
+		for(var/mob/living/L in get_turf(src))
+			if(L.stat || user == L)
+				continue
+			L.attack_hand(user)
+			user.changeNext_move(CLICK_CD_MELEE)
+			return
+	. = ..()
```

Comment:
```suggestion
	return ..()
```

Reply:
this is how the attackhand code is elsewhere and im scared its gonna break something esoteric if i change it
(also not a maintainer, i care not)